docs/cli.md

```diff
@@ -58,10 +58,13 @@ Output:
 
 ### Piping Example
 
-Ada can process URLs from piped input, making it easy to integrate with other command-line tools. Here's an example of how to pipe the output of another command into Ada. 
+Ada can process URLs from piped input, making it easy to integrate with other command-line tools. Here's an example of how to pipe the output of another command into Ada.
+
+Given a list of URLs, one by line, we may query the normalized URL string (`href`)
+and detect any malformed URL:
 
 ```bash
-cat dragonball_url.txt | ./buildbench/tools/adaparse
+cat dragonball_url.txt | adaparse -get href
```

Comment:
CLIs mostly support a single dash `-` if it's a shortcut. What about using `--get` and `-g`?

Reply:
It is already so. The documentation was in error. Fixing.